app/src/androidTest/java/com/sdp/movemeet/view/activity/ActivityDescriptionActivityTest.java

```diff
@@ -37,8 +31,6 @@
 import static androidx.test.espresso.action.ViewActions.click;
 import static androidx.test.espresso.action.ViewActions.scrollTo;
 import static androidx.test.espresso.assertion.ViewAssertions.matches;
-import static androidx.test.espresso.intent.Intents.intended;
-import static androidx.test.espresso.intent.matcher.IntentMatchers.hasComponent;
 import static androidx.test.espresso.matcher.ViewMatchers.withId;
 import static androidx.test.espresso.matcher.ViewMatchers.withText;
```

Comment:
Optimizing Imports is important if fact! We should do it systematically after editing a file, this will keep our work cleaner and more organised!